app/lib/features/events/widgets/add_event_location_widget.dart

```diff
@@ -38,22 +37,14 @@ class _AddEventLocationWidgetState extends ConsumerState<AddEventLocationWidget>
     _selectedType = widget.initialLocation?.type ?? LocationType.virtual;
 
     // Initialize editor states with initial data
-    if (widget.initialLocation == null) {
-      textEditorNoteState = EditorState.blank();
-      textEditorAddressState = EditorState.blank();
-    } else {
-      textEditorNoteState = EditorState(
-        document: ActerDocumentHelpers.parse(
```

Comment:
don't need markdown parsers also, as opted out in favor of html which ```copyMessageText``` can take care perfectly!